src/main/java/ca/on/oicr/pineryreports/reports/impl/StockReport.java

```diff
@@ -90,24 +93,25 @@ public void processOptions(CommandLine cmd) throws ParseException {
 
   @Override
   public String getTitle() {
-    return project
-        + " Stock Report, "
+    return " Stock Report for "
+        + String.join(", ", projects)
+        + ": "
         + (start == null ? "Any Time" : start)
         + " - "
         + (end == null ? "Now" : end);
   }
 
   @Override
   protected void collectData(PineryClient pinery) throws HttpResponseException {
-    List<SampleDto> allSamples = pinery.getSample().all();
+    List<SampleDto> allSamples =
+        pinery.getSample().allFiltered(new SamplesFilter().withProjects(projects));
```

Comment:
This probably shouldn't be filtered by project as the parents are needed and it's possible for parents to be in different projects

Reply:
Ah yes, PCSI...

Reply:
fixed